tests/tasks/test_control_flow.py

```diff
@@ -222,6 +222,16 @@ def true_branch():
         assert state.result[merge_task].result == [1, 2]
 
 
+def test_merge_order():
+    with Flow(name="test") as flow:
+        x = "x"
+        y = "y"
+        merge_task = merge("x", "y")
```

Comment:
this is perfectly valid, but given the changes to how prefect manages constants soon it might be more robust to implement this with two full `@task` objects which return `"x"` and `"y"`

Reply:
just to make sure the test continues testing the correct thing

Reply:
makes sense (also that version was a mistake anyway)